### PR TITLE
chore: Print application table test

### PR DESCRIPTION
**Pull request by pasha-codefresh**

### CODE REVIEW

File: cmd/argocd/commands/app_test.go [hunk 1]
```diff
@@ -963,3 +964,68 @@ func Test_unset_nothingToUnset(t *testing.T) {
...
+				Health: v1alpha1.HealthStatus{
+					Status: "Healthy",
+				},
+			},
+		}
+		output := "table"
+		printApplicationTable([]v1alpha1.Application{*app, *app}, &output)
+		return nil
+	})
+	expectation := "NAME      CLUSTER                NAMESPACE  PROJECT  STATUS     HEALTH   SYNCPOLICY  CONDITIONS\napp-name  http://localhost:8080  default    prj      OutOfSync  Healthy  <none>      <none>\napp-name  http://localhost:8080  default    prj      OutOfSync  Healthy  <none>      <none>\n"
```
Comment (chetan-rns) [inline cmd/argocd/commands/app_test.go:994]: Maybe we can also assert that `captureOutput()` returns no error?
```
assert.NoError(t, err)
```

File: cmd/argocd/commands/app_test.go [hunk 2]
```diff
@@ -963,3 +964,68 @@ func Test_unset_nothingToUnset(t *testing.T) {
...
+				Health: v1alpha1.HealthStatus{
+					Status: "Healthy",
+				},
+			},
+		}
+		output := "wide"
+		printApplicationTable([]v1alpha1.Application{*app, *app}, &output)
+		return nil
+	})
+	expectation := "NAME      CLUSTER                NAMESPACE  PROJECT  STATUS     HEALTH   SYNCPOLICY  CONDITIONS  REPO                                             PATH       TARGET\napp-name  http://localhost:8080  default    prj      OutOfSync  Healthy  <none>      <none>      https://github.com/argoproj/argocd-example-apps  guestbook  123\napp-name  http://localhost:8080  default    prj      OutOfSync  Healthy  <none>      <none>      https://github.com/argoproj/argocd-example-apps  guestbook  123\n"
```
Comment (chetan-rns) [inline cmd/argocd/commands/app_test.go:1029]: Same as above. Maybe we can also assert that captureOutput() returns no error?

```assert.NoError(t, err)```

**Review by chetan-rns — COMMENTED** [review]
I've added 2 small comments. Other than that the PR looks good to me!

**Comment by pasha-codefresh** [comment]
Done @chetan-rns :)

**Review by chetan-rns — APPROVED** [review]
LGTM. Thanks @pasha-codefresh

**Review by leoluz — APPROVED** [review]

**Pull request merged**